script/Arcueid/Day 8/08_00_ARC08_3.txt

```diff
@@ -217,22 +242,27 @@ What do I do? If I leave things as they are, what on Earth will happen to my sch
 [sha:5904a678670d55f3cf0e8bda7ab04a7e1b7c9860]{
 -- Page 155, Offset 20055.
 -- いや落ち着け、まだ大丈夫、問題ない。
-Calm down, it's still fine. There's no problem yet.
+No, calm down. Everything is okay. We don't have a problem just yet.
 }
 [sha:e3da3b5028af7d425e86fd29679df48ecb51e0e1]{
 -- Page 155, Offset 20056.
 -- 他の生徒は気づいていないし、隣の教室からも騒ぎは聞こえてこない。幸い、今はこの教室からしか見えない位置にいるようだ。
-The other students haven't noticed her, and I don't hear any uproar from the neighboring classrooms. Fortunately, it seems like this classroom is the only location from which she's visible right now.
+The other students haven't noticed her,
+ and I don't hear any commotion from the neighboring classrooms either.
+ Fortunately, she seems to be situated such that she's only visible from this classroom at the moment.
 }
 [sha:af6fd92b5d76b6f56ae4324bfc7163985d37b9bb]{
 -- Page 156, Offset 20057.
 -- “正真正銘、根っからの<破壊魔|デストロイヤー>め……！”
-"That woman is truly a destroyer from the very start...!"
+// Not voiced
+// Destroyer reads kinda whack but can't think of anything better rn and it is ruby en
```

Comment:
seeing that white woman cross the campus grounds, I remembered the line from the Hindu scripture. Vishnu is trying to persuade the Prince that he should follow through on his duty and to impress him takes on his multi-armed form.